### PR TITLE
bradl3yC - 8855 - 9383 - Debt letters mock response and redux

### PR DESCRIPTION
**Pull request by bradl3yC**
## Description
- [x] - Create functions to return mock data 
- [x] - Create initial redux actions and reducers for various states of retrieving debts data

## Testing done
- [x] - Added actions and reducers unit test

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by maharielrosario — APPROVED** [review]

**Review by U-DON — APPROVED** [review]
You can add this app to [the codeowners file](https://github.com/department-of-veterans-affairs/vets-website/blob/cbe3a1c194319defea7c40c3c5a05429159de11f/.github/CODEOWNERS) to avoid being blocked on reviews from the frontend-review-group.

**Pull request merged**
